### PR TITLE
feat(suite): Empty Coinjoin account screen

### LINKED ISSUE #6217: CoinJoin empty account
CJ empty account

### PR DESCRIPTION
**Pull request by dahaca**
## Description

Added empty coinjoin account screen

## Related Issue

Resolve [#6217](https://github.com/trezor/trezor-suite/issues/6217)

## Screenshots

image

### CODE REVIEW

File: packages/suite/src/views/wallet/transactions/components/CoinjoinAccountEmpty.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            <FeeText>
+                <Translation
+                    id="TR_COINJOIN_ACCESS_ACCOUNT_STEP_INITIAL_FEE_MESSAGE"
+                    values={{ fee: percentageFee }}
+                />
+            </FeeText>
+
+            <Button onClick={() => dispatch(goto('wallet-receive', { preserveParams: true }))}>
+                <Translation
+                    id="TR_RECEIVE_NETWORK"
```
Comment (komret) [inline packages/suite/src/views/wallet/transactions/components/CoinjoinAccountEmpty.tsx:61]: The design has "Receive funds". This change is reasonable, but has it been discussed?

File: packages/suite/src/views/wallet/transactions/components/CoinjoinAccountEmpty.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+    return (
+        <Container>
+            <Image image="COINJOIN_MESS" width={300} />
+
+            <Heading>
+                <Translation id="TR_COINJOIN_ACCESS_ACCOUNT_STEP_INITIAL_TITLE" />
+            </Heading>
+
+            <AccountDescriotion>
```
Comment (komret) [inline packages/suite/src/views/wallet/transactions/components/CoinjoinAccountEmpty.tsx]: typo

File: packages/suite/src/views/wallet/transactions/components/CoinjoinAccountEmpty.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+export const CoinjoinAccountEmpty = ({ account }: CoinjoinAccountEmptyProps) => {
+    const { percentageFee } = getCoinjoinConfig(account.symbol);
+
+    const dispatch = useDispatch();
+
+    return (
+        <Container>
+            <Image image="COINJOIN_MESS" width={300} />
```
Comment (komret) [inline packages/suite/src/views/wallet/transactions/components/CoinjoinAccountEmpty.tsx:42]: Not a fan of how the image looks in dark theme. Can we have a reversed variant of it (with dark background)?

**Review by komret — CHANGES_REQUESTED** [review]

File: packages/suite/src/views/wallet/transactions/components/CoinjoinAccountEmpty.tsx (hunk 1 above)
Comment (dahaca) [reply packages/suite/src/views/wallet/transactions/components/CoinjoinAccountEmpty.tsx:61]: I decided to use the same wording as in all other accounts :)

File: packages/suite/src/views/wallet/transactions/components/CoinjoinAccountEmpty.tsx (hunk 3 above)
Comment (dahaca) [reply packages/suite/src/views/wallet/transactions/components/CoinjoinAccountEmpty.tsx:42]: I can probably use `invert()` filter ;D

Comment (matejkriz) [reply packages/suite/src/views/wallet/transactions/components/CoinjoinAccountEmpty.tsx:42]: I believe the image is not final 🚧

File: packages/suite/src/views/wallet/transactions/components/CoinjoinAccountEmpty.tsx (hunk 2 above)
Comment (dahaca) [reply packages/suite/src/views/wallet/transactions/components/CoinjoinAccountEmpty.tsx]: b9651841a401e448bd1c7d05f695c6b104bbf621

File: packages/suite/src/views/wallet/transactions/components/CoinjoinAccountEmpty.tsx (hunk 3 above)
Comment (dahaca) [reply packages/suite/src/views/wallet/transactions/components/CoinjoinAccountEmpty.tsx:42]: Yes, confirmed by Ondra, already requested a better one.

**Review by komret — APPROVED** [review]

**Pull request merged**